blockchain/README.md

```diff
@@ -0,0 +1,41 @@
+## Create a new keypair
```

Comment:
also, we need to add prerequisits, and tell them how to install Docker on Windows or Linux easily